### ISSUE TITLE
[Suggestion]: Orders improvements

### ISSUE THREAD

**Issue by Lorith** [JGRennison/OpenTTD-patches#439]
A few things I've noticed would be very helpful while working on orders, arranged in suspected difficulty:

- Retarget conditional orders -- if I am doing a chain of orders that jump between each other, adding a single new order can require replacing multiple conditional orders throughout the list, depending on how it is set up. (In case anyone else does similar: deleting an existing target of a conditional order retargets to the order above for anything that targeted the deleted order, this can avoid having to redo the whole list)
- Move orders up/down in the list, similar to how you can in the programmable signals
- Copy order -- sometimes I like to have a dozen trains waiting in a depot and send them to a station only when there is a full load waiting, and it would be much easier if I could just copy a conditional order and set the via rather than having to set to waiting cargo amount, cargo type, less than, amount, and via when the only change is the via (and target of conditional jump)

**Comment by JGRennison** [comment]
Move orders up/down in the list: This is a vanilla feature, you can drag and drop orders to move them within the order list

The other two should not be technically difficult, the main issue with these sorts of things is where to add an extra button or UI mode.

**Comment by Lorith** [comment]
Well, that is embarrassing.  I know I tried drag and drop, but it didn't work.  Maybe I should have tried more than once in case I accidentally let go of the button too early or something, because it absolutely did work when I just tried it again.

And maybe shift-click and alt-click would work?  I saw in the hover-tip that ctrl-click zooms to target, but neither of the others seem to have an effect besides that of a normal click.
